workspaces_route_lock.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  workspaces_route_lock.manager:
+    class: Drupal\workspaces_route_lock\RouteLockedWorkspacesManager
+    decorates: 'workspaces.manager'
+    decoration_priority: -5
```

Comment:
Why `-5`?

Reply:
The lower the value the later it will be applied. In this case we want to go later than the default (which is 0).